pkg/controller/podautoscaler/podautoscaler_controller.go

```diff
@@ -1080,6 +1081,15 @@ func (r *PodAutoscalerReconciler) computeMetricBasedReplicas(
 		return nil, fmt.Errorf("failed to get pod selector: %w", err)
 	}
 
+	req, err := labels.NewRequirement(constants.PodGroupIndexLabelKey, selection.Equals, []string{"0"})
```

Comment:
em. there's a problem. 

If PodGroupSize is not being used, this label won't exist. If we can the labelSelector here. it will fail the normal case?

Reply:
I see, will fix that.